classes/classes/Scenes/Areas/Bog/ChameleonGirlScene.as

```diff
@@ -303,25 +303,6 @@ package classes.Scenes.Areas.Bog
 			combat.cleanupAfterCombat();
 		}
 
-		//Item Use Scenes Intro (Victory) (Z edited)
-		private function useAnItemOnTheChamcham():void
```

Comment:
This function was left unused after menu refactoring in one of the previous commits. So I guess they are related.